plugins/knife/knife-security-groups-create.rb

```diff
@@ -50,7 +50,9 @@ class SecurityGroupsCreate < Chef::Knife
     end
 
     def run
-      commission_security_groups(clearwater_security_groups,
+      extra_groups = attributes["extra_groups"] rescue {}
```

Comment:
`extra_groups` is `internal-sip`-specific, either name it as such or make it generic.